getting-started/jdbc/docker-compose.yml

```diff
@@ -32,6 +32,9 @@ services:
       - JAVA_DEBUG=true
       - JAVA_DEBUG_PORT=*:5005
       - POLARIS_PERSISTENCE_TYPE=relational-jdbc
+      - POLARIS_PERSISTENCE_RELATIONAL_JDBC_MAX_RETRIES=5
+      - POLARIS_PERSISTENCE_RELATIONAL_JDBC_INITIAL_DELAY_IN_MS=100
+      - POLARIS_PERSISTENCE_RELATIONAL_JDBC_MAX_DELAY_IN_MS=5000
```

Comment:
nit: Don't defaults work for "getting started"?

Reply:
They do ! these are just for demo purpose !